src/layout.rs

```diff
@@ -5,24 +5,22 @@ use leptos_router::{Outlet, RoutingProgress};
 
 #[component]
 pub fn Layout(is_routing: ReadSignal<bool>) -> impl IntoView {
+  let ui_theme = expect_context::<RwSignal<String>>();
```

Comment:
I imagine this will be part of the site response down the line.

Reply:
yeah, good point, an instance could have default theme in future 

this signal will carry that theme identifier to the part of the page where it needs to be set